### ISSUE TITLE
Shaderconv doesn't create _gl4es_ attributes when ftransform() is translated

### ISSUE THREAD

**Issue by Icenowy** [ptitSeb/gl4es#33]

**Comment by Icenowy** [comment]
I did a ugly hack which adds

```
"uniform highp mat4 _gl4es_ModelViewProjectionMatrix;\n"
"attribute highp vec4 _gl4es_Vertex;\n"
```

to gl4es_ftransformSource, the game Taisei now fully works (it mixes fixed pipeline and shader-based rendering).

Without this fix the stage is not correctly cleared.

Is this hack suitable?

**Comment by ptitSeb** [comment]
Oh, I see.

I'll think of something to avoid double declaration of those 2 variables.

**Comment by ptitSeb** [comment]
It should be fixed with 70e7f50dcb47ce5e9ed9c3965867f5f76d7827cf , if you can try.

**Comment by Icenowy** [comment]
Fixed, thanks!
